src/GraphQL/Types/Composite/AutoRegisteringHelper.cs

```diff
@@ -42,21 +41,34 @@ internal static FieldType CreateField(MemberInfo memberInfo, Type graphType, boo
                 DeprecationReason = memberInfo.ObsoleteMessage(),
                 Type = graphType,
                 DefaultValue = isInputType ? memberInfo.DefaultValue() : null,
-                Resolver = isInputType ? null : MemberResolver.Create(memberInfo),
             };
             if (isInputType)
             {
                 fieldType.WithMetadata(ComplexGraphType<object>.ORIGINAL_EXPRESSION_PROPERTY_NAME, memberInfo.Name);
             }
+            if (!isInputType &&
+                memberInfo is MethodInfo methodInfo &&
+                fieldType.Name.EndsWith("Async") &&
+                methodInfo.ReturnType.IsGenericType &&
+                methodInfo.ReturnType.GetGenericTypeDefinition() == typeof(Task<>))
```

Comment:
\+ ValueTask<> support?

Reply:
As of now there is no ValueTask support for the entire GraphQL.NET execution pipeline. It is something I’d like to explore. It would apply to the entire solution at once, not just this method, I expect.

Reply:
ok